### PR TITLE
Use Terraform locals to avoid variable duplication

### PR DESCRIPTION
**Pull request by johananl**
I'm opening this PR after multiple times I've made copy-paste mistakes when deploying clusters. I think this can be solved nicely by using [Terraform locals](https://www.terraform.io/docs/configuration/locals.html).

### CODE REVIEW

**Review by rata — APPROVED** [review]
LGTM

I was already using locals on my terraform file, didn't realize docs weren't. Good catch! :)

File: docs/flatcar-linux/packet.md [hunk 1]
```diff
@@ -109,6 +109,12 @@ Additional configuration options are describe in the `packet` provider [docs](ht
 Define a Kubernetes cluster using the controller module [packet/flatcar-linux/kubernetes](https://github.com/kinvolk/lokomotive-kubernetes/tree/master/packet/flatcar-linux/kubernetes) and the worker module [packet/flatcar-linux/kubernetes/workers](https://github.com/kinvolk/lokomotive-kubernetes/tree/master/packet/flatcar-linux/kubernetes).
 
 ```tf
+locals {
+  project_id = "93fake81..."
+  cluster_name = "supernova"
+  facility = "ams1"
+}
```
Comment (surajssd) [inline docs/flatcar-linux/packet.md:116]: can we also do the same for providers?

**Review by surajssd — CHANGES_REQUESTED** [review]

**Review by indradhanush — APPROVED** [review]
I've had the same problem. :100: for helping fix it!

LGTM.

**Review by invidian — APPROVED** [review]

**Comment by invidian** [comment]
@johananl can you please follow up here so we can merge it?

File: docs/flatcar-linux/packet.md (hunk 1 above)
Comment (johananl) [reply docs/flatcar-linux/packet.md:116]: Doesn't seem to be possible.

File: docs/flatcar-linux/packet.md [hunk 2]
```diff
@@ -109,6 +109,12 @@ Additional configuration options are describe in the `packet` provider [docs](ht
 Define a Kubernetes cluster using the controller module [packet/flatcar-linux/kubernetes](https://github.com/kinvolk/lokomotive-kubernetes/tree/master/packet/flatcar-linux/kubernetes) and the worker module [packet/flatcar-linux/kubernetes/workers](https://github.com/kinvolk/lokomotive-kubernetes/tree/master/packet/flatcar-linux/kubernetes/workers).
 
 ```tf
+locals {
+  project_id = "93fake81..."
```
Comment (invidian) [inline docs/flatcar-linux/packet.md]: Should we align those locals here?

**Pull request merged**